.buildkite/scripts/Dockerfile-arm32

```diff
@@ -0,0 +1,41 @@
+FROM arm32v7/erlang:22.3.3-alpine as builder
+
+RUN apk add --no-cache --update \
+    git tar build-base linux-headers autoconf automake libtool pkgconfig \
+    dbus-dev bzip2 bison flex gmp-dev cmake lz4 libsodium-dev openssl-dev \
+    sed wget
+
+WORKDIR /usr/src/miner
+
+# The no-psabi flag comes from:
+# https://stackoverflow.com/a/48149400
+ENV CC=gcc CXX=g++ CFLAGS="-U__sun__ -Wno-psabi" \
+    ERLANG_ROCKSDB_OPTS="-DWITH_BUNDLE_SNAPPY=ON -DWITH_BUNDLE_LZ4=ON" \
+    ERL_COMPILER_OPTIONS="[deterministic]"
+
+# Add our code
+ADD . /usr/src/miner/
+
+RUN rebar3 as prod tar
+RUN mkdir -p /opt/rel
+RUN tar -zxvf _build/prod/rel/*/*.tar.gz -C /opt/rel
+RUN sed -i 's/^\s*{key,/%{key,/' /opt/rel/releases/0.1.0/sys.config
+RUN sed -i 's/{use_ebus, true},/{use_ebus, false},/' /opt/rel/releases/0.1.0/sys.config
+RUN sed -i 's/{radio_device, { {127,0,0,1}, 1680,/{radio_device, { {0,0,0,0}, 1680,/' /opt/rel/releases/0.1.0/sys.config
+RUN mkdir -p /opt/rel/update
+RUN wget https://github.com/helium/blockchain-api/raw/master/priv/prod/genesis
+RUN cp genesis /opt/rel/update/genesis
+
+FROM arm32v7/erlang:22.3.3-alpine as runner
+
+RUN apk add --no-cache --update ncurses dbus gmp libsodium gcc
+RUN ulimit -n 64000
+
+WORKDIR /opt/miner
+
+ENV COOKIE=miner \
+    # Write files generated during startup to /tmp
+    RELX_OUT_FILE_PATH=/tmp
+
+COPY --from=builder /opt/rel /opt/miner
+
```

Comment:
Same question? Does cleaning up a lot of the source and build output help make the image slimmer?

Reply:
Yes it does